### ISSUE TITLE
Wrangle, and nail down correct metric/unit for domContentFlushed

### ISSUE THREAD

**Issue by stephendonner** [mozilla/wpt-api#97]
tl;dr: what's the magic to go from a milliseconds-sinceUNIX-Epoch-Time ```domContentFlushed``` (1539735909403) to whichever metric name, type, and unit (assuming milliseconds?) we want to measure + visualize?

We need to have a vetted formula/recipe for the nascent, raw ```domContentFlushed``` metric, which I landed here in https://github.com/mozilla/wpt-api/pull/79, and have been submitted to, and visualizing in, DataDog: https://app.datadoghq.com/dash/879449/youtube-googlecom-facebook-perf-tests-fx-relnightly-chrome-relcanary?live=true&page=0&is_auto=false&from_ts=1537213887340&to_ts=1539805887340&tile_size=m

See https://wpt.stage.mozaws.net/result/181017_8H_R/?f=json (grep for "flush") - you'll find it under:
data > median > firstView > custom > domContentFlushed, with a value of **```1539735909403```**, which is milliseconds since UNIX Epoch Time (https://www.epochconverter.com/):
```
GMT: Wednesday, October 17, 2018 12:25:09.403 AM
```

**Comment by stephendonner** [comment]
We're going to go with ```timeToDOMContentFlushed``` ```-``` ```fetchStart```.

See https://wpt.stage.mozaws.net/result/181023_6Y_6J/?f=json (though it won't be around for long/archived).  From that:

```timeToDOMContentFlushed: 1540325541694, fetchStart: 1540325538950```

So, timeToDOMContentFlushed - fetchStart = 2744 ms == 2744 / 1000 = 2.74 seconds

**Comment by stephendonner** [comment]
I've put up a PR over in https://github.com/mozilla-services/webpagetest-settings/pull/12 to get ```fetchStart``` exposed to WebPageTest, and then we'll need to make a Python function to do the simple math of ```timeToDOMContentFlushed - fetchStart```, and store/publish the offset as ```domContentFlushed```.

**Comment by stephendonner** [comment]
Tonight, I also discovered that ```start_time```, which is on the core ```webpagetest``` server/API side, looks like this, in the Python code, in case this is also useful in determining/calculating this or other metrics, from WebPageTest [0]:

```
def wait_for_page_load(self):
        """Wait for the onload event from the extension"""
       if self.job['message_server'] is not None and self.connected:
start_time = monotonic.monotonic()
```

[0] https://github.com/WPO-Foundation/wptagent/blob/11222c7ab48bafb1203494dc4089fa298e75e040/internal/firefox.py#L327-L330(wptagent/internal/firefox.py
